node/helpers.go

```
@@ -135,10 +133,21 @@
 
 func waitUntilBlockHashSeen(node Node, blockHash string) error {
-	return testutils.Retry(300, 100*time.Millisecond, func() error {
+	return Retry(300, 100*time.Millisecond, func() error {
 		if isBlockHashSeen(node, blockHash) {
 			return nil
 		}
 		return fmt.Errorf("block hash %v not seen", blockHash)
 	})
 }
+
+func Retry(tries int, durationBetweenAttempts time.Duration, fn func() error) (err error) {
```

golangci-lint (1.62.2)

[warning] exported: exported function Retry should have comment or be unexported

(revive)